features/api/metrics/delete.feature

```diff
@@ -30,3 +30,12 @@ Feature: Product > Integration > Metrics > Edit
     Given I change to tab "Metrics"
     When I follow "Hits"
     Then I should not see "Delete"
+
+  Scenario: Cannot delete a metric used in the latest gateway configuration
+    Given metric "Pasta" is mapped
+    And the gateway configuration is promoted to staging
+    When I change to tab "Metrics"
+    And I follow "Pasta"
+    And I press "Delete" and I confirm dialog box
+    Then I should see the flash message "Metric is used by the latest gateway configuration and cannot be deleted"
+    And I should see "Pasta" on the metrics tab
```

Comment:
```suggestion
    Then I should see then following metrics
      | Pasta |
```

Reply:
`the`?

Reply:
I didn't like this because in this case we need to list all metrics, in this case including "Hits", but there is no mention of "Hits" in the scenario, and we shouldn't care about any other metrics present, just care about the one we're trying to delete.

But whatever.